setup.cfg

```diff
@@ -1,3 +1,41 @@
 [wheel]
-universal=1
+universal = 1
+
+[metadata]
+name = widgetastic.core
+author = Milan Falesnik
```

Comment:
We need to add maintainers, Milan no longer works on this

Reply:
Agreed, will do.

Reply:
If you wanty, you can remove me from the `author` field and just add some "collective" in there, definitely for the e-mail (which is no longer valid), but I may start using this lib in my job again therefore possibly making contributions again.

Reply:
I against that. The original author should stay here at least the name.